bin/heudiconv

```diff
@@ -1060,6 +1061,13 @@ def save_scans_key(item, bids_files):
         f_name = f_name.replace('json', 'nii.gz')
         rows[f_name] = get_formatted_scans_key_row(item)
         subj_, ses_ = _find_subj_ses(f_name)
+        if not subj_:
```

Comment:
is this a sufficient check for bids?

Reply:
@yarikoptic ping here :)

Reply:
checked "upstairs" -- there is `is_bids` check upon calling this function, so should be kosher I guess